### DISCUSSION TITLE
Magikarp-style equipment

### DISCUSSION THREAD

**Discussion by doubledh** [Imaginary-Horizons-Productions/prophets-of-the-labyrinth-Archive#434] [Ideas]
I want an equipment that is impractical in its first form, but after upgrading (either multiple layers, or only after using up most/all of it) it becomes an apex piece of equipment.

I'm thinking possibly a literal fish-of-a-club that evolves into a shark-of-a-club or something. You ever seen Maplestory's Frozen Tuna? https://uncyclopedia.com/w/images/e/e6/Mapletuna.PNG

**Comment by ntseng** [comment]
I was thinking about something similar to this, except along crit lines instead of upgrades: Splash on normal hit, Outrage on crit.

Also if we think of ways to make upgrade progression go longer, we can pull off Onion Knight vibes.
